src/services/resource-service.ts

```diff
@@ -7,7 +7,7 @@ import { URLs } from '~/constants/request'
 import {
```

Comment:
Check this, probably all of the imports are just types
```suggestion
import type {
```